x-pack/plugins/search_playground/public/hooks/use_load_connectors.ts

```diff
@@ -39,7 +40,8 @@ const connectorTypeToLLM: Array<{
     actionProvider: OpenAiProviderType.AzureAi,
     match: (connector) =>
       connector.actionTypeId === OPENAI_CONNECTOR_ID &&
-      (connector as OpenAIConnector).config.apiProvider === OpenAiProviderType.AzureAi,
+      (connector as OpenAIConnector)?.config !== null &&
```

Comment:
`(connector as OpenAIConnector)?.config` is `undefined` when using a preconfigured connector and the error blocked me from moving past the first page:
<img width="1351" alt="Screenshot 2024-06-25 at 3 54 43 PM" src="https://github.com/elastic/kibana/assets/6935300/326d53bb-9064-4f2a-a702-d6710fea9c0d">